src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithAllFeatures.cs

```diff
@@ -79,6 +79,7 @@ public void It_publishes_the_project_correctly(string targetFramework, string []
             ""System.Runtime.TieredCompilation.QuickJitForLoops"": true,
             ""System.StartupHookProvider.IsSupported"": false,
             ""System.Resources.ResourceManager.AllowCustomResourceTypes"": false,
+            ""System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization"": false,
```

Comment:
I've got a question here. Why are all the features seemingly turned off here when the file is named `GivenThatWeWantToPublishAProjectWithAllFeatures.cs`?

Reply:
They aren't all turned off. Some are set to `true`, a lot are set to `false`. The point of this test is to ensure that setting an MSBuild property sets the correct runtimeconfig switch. What specific value the properties are set to doesn't matter as much as that they match what the `KitchenSink/TestApp/TestApp.csproj` said they should be.